JetSubStructureUtils/Root/BoostedXbbTag.cxx

```diff
@@ -545,10 +589,12 @@ std::vector<const xAOD::Jet*> BoostedXbbTag::get_bTagged_trackJets(const xAOD::J
   std::vector<const xAOD::Jet*> associated_trackJets;
   // get the track jets from the parent
   (*parent(jet))->getAssociatedObjects<xAOD::Jet>("GhostAntiKt2TrackJet", associated_trackJets);
+  //// IF PROBLEMS with association to ungroomed parent, comment out to get track jets from trimmed jet
+  //jet.getAssociatedObjects<xAOD::Jet>("GhostAntiKt2TrackJet", associated_trackJets);
```

Comment:
This should be set as a configuration in the function rather than having the user comment it out or something.